server/settings.js

```diff
@@ -5,7 +5,8 @@ const defaultWindows = {
     attackersDeclared: true,
     defendersDeclared: true,
     dominance: false,
-    standing: false
+    standing: false,
+    taxation: false
```

Comment:
This changes the current behavior, which was always have a psuedo-window w/ the End Round prompt. Now by default it'll skip from the Challenges phase right back to the Plot phase with no pause (if they don't need to discard for reserve), which is a significant flow / tempo change. Are you sure this is what we want?

Reply:
Yes.  The current end round prompt isn't in the game rules and adds nothing that the new window doesn't give instead.  Sure there'll be some annoyed people during the transition, but when isn't there..